### PR TITLE
Fix for #1731

### LINKED ISSUE #1731: Fix Cancele Delete Settings Field
- Laravel Version: 5.4.*
- Voyager Version: 0.11.14
- PHP Version: 5.6.4
- Database Driver & Version:

### Description:
```
<form action="http://laravel/admin/settings/???" <- id not updating id="delete_form" method="POST">
                        <input type="hidden" name="_method" value="DELETE">
                        <input type="hidden" name="_token" value="Rj0CIfoJOVPHmiV9n8eZ5CUj3jiBf4tD8b7GBgkY">
                        <input type="submit" class="btn btn-danger pull-right delete-confirm" value="Yes, Delete This Setting">
                    </form>
```
### Steps To Reproduce:
1.Cancele Delete Setting Item
2.Select Next setting field and delete
3. Id  of the deleted field is not updated

### PR DESCRIPTION
**Pull request by vblinden**
This fixes #1731. The setting id are now correctly updated everytime on the delete form.

Thanks to @davodavodavo3 for the orginal pull request (#1764)!

Closes #1764

### CODE REVIEW

File: resources/views/settings/index.blade.php [hunk 1]
```diff
@@ -444,10 +444,12 @@
                 }
             });
 
-            $('.voyager-trash').click(function () {
+            $('.panel-actions .voyager-trash').click(function () {
```
Comment (fletch3555) [inline resources/views/settings/index.blade.php:449]: What's this class for?

File: resources/views/settings/index.blade.php [hunk 2]
```diff
@@ -444,10 +444,12 @@
...
                 var display = $(this).data('display-name') + '/' + $(this).data('display-key');
 
                 $('#delete_setting_title').text(display);
+
+                $('#delete_form')[0].action = '{{ route('voyager.settings.delete', [ 'id' => '__id' ]) }}';
```
Comment (fletch3555) [inline resources/views/settings/index.blade.php]: Can't we combine these 2 lines and only have to set the action once? Something like:
```
$('#delete_form')[0].action = '{{ route('voyager.settings.delete', [ 'id' => '__id' ]) }}'.replace('__id', $(this).data('id'));
```

If this works, there are other instances of this that should be updated, but that can be a separate PR since it's unrelated to #1731

This also renders the form action useless.  Can you get rid of the unnecessary `route()` call and replace it with `#` or something instead?

**Review by fletch3555 — CHANGES_REQUESTED** [review]

File: resources/views/settings/index.blade.php (hunk 1 above)
Comment (vblinden) [reply resources/views/settings/index.blade.php:449]: This fixes the error when a user clicked the trash icon in the modal the value went: undefined/undefined. This ensures that only the trash icons in the actions are used to get the data-id.

**Review by fletch3555 — APPROVED** [review]
